src/helpers/fetchEndpointWithCorrelationId.test.js

```diff
@@ -0,0 +1,33 @@
+import fetchEndpointWithCorrelationId from "../helpers/fetchEndpointWithCorrelationId";
+import { v4 as uuidv4 } from "uuid";
+import nock from "nock";
+
+describe("fetchEndpointWithCorrelationId", () => {
+  let body = JSON.stringify({ code: "code" });
+  let correlationId = `${uuidv4()}-login`;
+  let endpoint = "/api/session";
+  let method = "POST";
+  let scope;
+
+  beforeEach(() => {
+    scope = nock("http://localhost:3001", {
```

Comment:
We should probably have the target API set by an environment variable so we don't have to change it in multiple places

Reply:
yeah agreed, I think its only being used in one other place for now.

Reply:
yeah i'd leave it as is ... when we want to change it then you have validated your hypothesis that it should be external, change it then.